sdk/keyvault/keyvault-certificates/src/generated/keyVaultClientContext.ts

```diff
@@ -10,7 +10,7 @@ import * as coreHttp from "@azure/core-http";
 import { ApiVersion73, KeyVaultClientOptionalParams } from "./models";
 
 const packageName = "@azure/keyvault-certificates";
-export const packageVersion = "4.5.0-beta.1";
+const packageVersion = "4.5.0-beta.1";
```

Comment:
In core v1 it _is_ emitted and used below, it's just not _exported_ - that was the manual step we had to do after regenerating code